platform/windows/os_windows.cpp

```diff
@@ -1000,11 +1015,10 @@ Error OS_Windows::execute(const String &p_path, const List<String> &p_arguments,
 
 	ProcessInfo pi;
 	ZeroMemory(&pi.si, sizeof(pi.si));
-	pi.si.cb = sizeof(pi.si);
+	pi.si.StartupInfo.cb = sizeof(pi.si);
 	ZeroMemory(&pi.pi, sizeof(pi.pi));
-	LPSTARTUPINFOW si_w = (LPSTARTUPINFOW)&pi.si;
+	LPSTARTUPINFOW si_w = (LPSTARTUPINFOW)&pi.si.StartupInfo;
 
-	bool inherit_handles = false;
```

Comment:
Not sure why `inherit_handles` was removed, it `execute` is called w/o pipe (only possible from C++ code), it will end up with uninitialized `lpAttributeList` and `bInheritHandles = True`.

It probably should be preserved, passed to `CreateProcessW` and used to set `creation_flags` to `NORMAL_PRIORITY_CLASS` only if not inheritance needed at all.

Reply:
You're absolutely right. Fixed.